src/utils/lambda.ts

```diff
@@ -0,0 +1,13 @@
+import { HttpResponse } from "../types/types";
+
+export const generateHttpResponse = <T>(statusCode: number, response: T): HttpResponse => {
```

Comment:
You can use not your custom type, but generic type from `aws-lambda` package

Reply:
Done